release/cc/execute_targets.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module to exec cc toolchain targets and extract contents for all configs."""
+
+from __future__ import print_function
+
+import atexit
+import os
+import shlex
+import shutil
+import subprocess
+import tarfile
+
+from util import get_autoconfig_target_name
+from util import get_git_root
+
+GIT_ROOT = get_git_root()
+CONFIG_REPO = "local_config_cc"
+CONFIG_FILES = ["CROSSTOOL", "BUILD", "cc_wrapper.sh", "dummy_toolchain.bzl"]
+TMP_DIR = os.path.join(GIT_ROOT, "release", "tmp")
+
+
+def _cleanup():
+  """Cleanup generated files."""
+  if os.path.isdir(TMP_DIR):
```

Comment:
Should't you use os.path.exists? Otherwise it doesn't nothing if TMP_DIR points to a file?

Reply:
Good point! Fixed.